src/main/java/com/matt/forgehax/util/command/Command.java

```diff
@@ -241,8 +242,9 @@ protected boolean processMain(ExecuteData data)
 
   protected boolean processChildren(@Nonnull String[] args)
       throws CommandExecuteException, NullPointerException {
-    if (args.length > 0) {
-      final String lookup = (args[0] != null ? args[0] : Strings.EMPTY).toLowerCase();
+    if (args.length > 0
+        && !args[0].startsWith(ChatCommandService.getActivationCharacter().toString())) {
```

Comment:
not sure if I am on board with this. what exactly is the issue?

Reply:
nevermind, I get it now. just seems like a bad idea because it is really only for this command and could break other mods